client/post-editor/editor-gutenberg-opt-in-dialog/index.jsx

```diff
@@ -122,15 +122,13 @@ const mapDispatchToProps = dispatch => ( {
 			withAnalytics(
 				composeAnalytics(
 					recordGoogleEvent(
-						'Gutenberg Opt-Out',
+						'Gutenberg Calypso Opt-In Dialog',
```

Comment:
Any reason for using here a different category from the used [in the line 106](https://github.com/Automattic/wp-calypso/pull/28525/files#diff-18994bb0e120de0f9cdc52e3f49d9536R106)?